tgui/packages/tgui/interfaces/AntagInfoGangmember.tsx

```diff
@@ -0,0 +1,137 @@
+import { useBackend } from '../backend';
+import { BlockQuote, Divider, Icon, Section, Stack } from '../components';
+import { Window } from '../layouts';
+
+type Info = {
+  antag_name: string;
+  gang_name: string;
+  gang_objective: string;
+  gang_clothes: string[];
+};
+
+export const AntagInfoGangmember = (props, context) => {
+  const { data } = useBackend<Info>(context);
+  const {
+    gang_name,
+    antag_name,
+  } = data;
+  return (
+    <Window
+      width={620}
+      height={500}>
+      <Window.Content
+        style={{
+          'background-image': 'none',
+        }}>
+        <Section fill>
+          <Stack vertical>
+            <Stack.Item textColor="red" fontSize="20px">
+              {gang_name} for life! You are a {antag_name}!
+            </Stack.Item>
+            <Stack.Item fontSize="18px">
+              As a gang member, support your family above all!
+              Tag turf with a spraycan, wear your family&apos;s clothes,
+              induct new members with induction packages,
+              and accomplish your family objective.
+            </Stack.Item>
+            <Divider />
+            <Stack.Item>
+              <Stack>
+                <Stack.Item width="50%">
+                  <GangClothesPrintout />
+                </Stack.Item>
+                <Stack.Item width="50%">
+                  <GangPhonePrintout />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+            <Divider />
```

Comment:
```suggestion
            <Stack.Divider />
```

Reply:
I used two dividers, so if you get this one be sure to get the one a few lines up.